src/intervals/special.jl

```diff
@@ -83,3 +83,11 @@ contains_zero(X::Interval{T}) where {T} = zero(T) ∈ X
 # """
 # widen(x::Interval{T}) where T<:AbstractFloat =
 #     Interval(prevfloat(x.lo), nextfloat(x.hi))
+
+"""
+    zero_interval(T)
+
+Return the zero interval `Interval(0, 0)` in the numeric type `T`.
+"""
+zero_interval(::Type{T}) where T<:Real = zero(Interval{T})
+zero_interval(x::Interval{T}) where T<:Real = zero(Interval{T})
```

Comment:
Note that there exists `zero` which does precisely this; see [src/intervals/arithmetic.jl](https://github.com/JuliaIntervals/IntervalArithmetic.jl/blob/b70940dabdbefd8eb4be32d7bde6df72174dd3f7/src/intervals/arithmetic.jl#L55).

Reply:
true. so do you suggest to remove `zero_interval(Float64)` (since it can be achieved by `zero(Interval{Float64})`?